packages/test/test-end-to-end-tests/src/test/opReentrancy.spec.ts

```diff
@@ -238,6 +239,52 @@ describeNoCompat("Concurrent op processing via DDS event handlers", (getTestObje
 				sharedDirectory2.getSubDirectory(topLevel)?.getSubDirectory(innerLevel)?.get(key),
 			);
 		});
+
+		it.only(`Eventual consistency for shared directories with op reentry and disconnect - ${
+			enableGroupedBatching ? "Grouped" : "Regular"
+		} batches`, async function () {
+			if (provider.driver.type === "t9s" || provider.driver.type === "tinylicious") {
+				// This test is flaky on Tinylicious. ADO:5010
+				this.skip();
+			}
+
+			await setupContainers({
+				...testContainerConfig,
+				runtimeOptions: {
+					enableGroupedBatching,
+				},
```

Comment:
do you also need to set `flushMode: FlushMode.TurnBased`?

Reply:
`TurnBased` is the default.